### PR TITLE
#185172086   User should Signup to the E-commerce App 

### PR DESCRIPTION
**Pull request by Pacific911**
#### What does this PR do?
- Users should be able to Signup and or Register using Sign up form on the signup route.

#### Description of Task to be completed?
 - A user should register with the information to access the e commerce website services

1. Below is the end point used to test

 - POST /accounts/signup

#### What are the relevant pivotal tracker/Trello stories?
#185172086

#### Screenshots (if appropriate)
register

### CODE REVIEW

**Review by Edcreation — APPROVED** [review]
Why those .min.map?

**Review by JosephNgabo — APPROVED** [review]

**Review by princoo — APPROVED** [review]

**Review by munezerobagira — COMMENTED** [review]
I am getting this error from the backend but the toast is showing the undefined
image

{
    "code": 406,
    "error": "Password must have 8 characters, Uppercase and Lowercase, a number and a Symbol"
}

File: vite.config.ts.timestamp-1685556800528-e1c6bed7431ef.mjs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
...aW5lQ29uZmlnIH0gZnJvbSAndml0ZSc7XG5pbXBvcnQgcmVhY3QgZnJvbSAnQHZpdGVqcy9wbHVnaW4tcmVhY3QnO1xuXG5leHBvcnQgZGVmYXVsdCBkZWZpbmVDb25maWcoe1xuICBwbHVnaW5zOiBbcmVhY3QoKV0sXG4gIHRlc3Q6IHtcbiAgICBnbG9iYWxzOiB0cnVlLFxuICAgIGVudmlyb25tZW50OiAnanNkb20nLFxuICAgIHNldHVwRmlsZXM6ICcuL3NyYy9zZXR1cFRlc3RzLnRzeCcsXG4gICAgY292ZXJhZ2U6IHtcbiAgICAgIHByb3ZpZGVyOiAnaXN0YW5idWwnLFxuICAgICAgcmVwb3J0ZXI6IFsndGV4dCcsICdqc29uJywgJ2h0bWwnLCAnbGNvdiddLFxuICAgICAgbGluZXM6IDgwLFxuICAgICAgZnVuY3Rpb25zOiA4MCxcbiAgICAgIGJyYW5jaGVzOiA4MCxcbiAgICAgIHN0YXRlbWVudHM6IDgwLFxuICAgICAgYWxsOiB0cnVlLFxuICAgICAgZXhjbHVkZTogWycqKi8qLnN0b3J5Ym9vaycsICcqKi8qLnN0b3JpZXMue3RzLHRzeH0nXSxcbiAgICB9LFxuICB9LFxufSk7XG4iXSwKICAibWFwcGluZ3MiOiAiO0FBQ0EsU0FBUyxvQkFBb0I7QUFDN0IsT0FBTyxXQUFXO0FBRWxCLElBQU8sc0JBQVEsYUFBYTtBQUFBLEVBQzFCLFNBQVMsQ0FBQyxNQUFNLENBQUM7QUFBQSxFQUNqQixNQUFNO0FBQUEsSUFDSixTQUFTO0FBQUEsSUFDVCxhQUFhO0FBQUEsSUFDYixZQUFZO0FBQUEsSUFDWixVQUFVO0FBQUEsTUFDUixVQUFVO0FBQUEsTUFDVixVQUFVLENBQUMsUUFBUSxRQUFRLFFBQVEsTUFBTTtBQUFBLE1BQ3pDLE9BQU87QUFBQSxNQUNQLFdBQVc7QUFBQSxNQUNYLFVBQVU7QUFBQSxNQUNWLFlBQVk7QUFBQSxNQUNaLEtBQUs7QUFBQSxNQUNMLFNBQVMsQ0FBQyxrQkFBa0IsdUJBQXVCO0FBQUEsSUFDckQ7QUFBQSxFQUNGO0FBQ0YsQ0FBQzsiLAogICJuYW1lcyI6IFtdCn0K
```
Comment (munezerobagira) [inline vite.config.ts.timestamp-1685556800528-e1c6bed7431ef.mjs:25]: why did you add  this file

Comment (Pacific911) [reply vite.config.ts.timestamp-1685556800528-e1c6bed7431ef.mjs]: requested changes solved 👍

**Comment by munezerobagira** [comment]
Have you verified that the issue is resolved

**Review by munezerobagira — APPROVED** [review]
